rpc/sigpipe.go

```diff
@@ -0,0 +1,9 @@
+// +build !darwin
+
+package rpc
+
+import "net"
+
+func DisableSigPipe(c net.Conn) error {
+	return nil
```

Comment:
Should this panic or return an "unsupported" error instead?  We shouldn't just ignore an option the user passed in and let them believe we're honoring it, right?